src/messaging/ExchangeMgr.cpp

```diff
@@ -202,9 +202,9 @@ void ExchangeManager::OnMessageReceived(const PacketHeader & packetHeader, const
     CHIP_ERROR err                          = CHIP_NO_ERROR;
     UnsolicitedMessageHandler * matchingUMH = nullptr;
 
-    ChipLogProgress(ExchangeManager, "Received message of type %d and protocolId %" PRIu32 " on exchange %d",
-                    payloadHeader.GetMessageType(), payloadHeader.GetProtocolID().ToFullyQualifiedSpecForm(),
-                    payloadHeader.GetExchangeID());
+    ChipLogProgress(ExchangeManager, "Received message of type 0x%02x with vendorId 0x%04x and protocolId 0x%04x on exchange %d",
```

Comment:
Please replace `PRIu32` with `PRIx32` and make sure otherwise that format specifiers are correct for any %d converted to hex, as the underlying type assumption will go from signed to unsigned.

Reply:
Resolving; these bits all seem to be correct.